### ISSUE TITLE
False positives makes esbuild to fail?

### ISSUE THREAD

**Issue by pateketrueke** [bluwy/publint#17]
I found that you MUST prefix the paths with `./` or they'll fail with esbuild, not sure if this happens with other bundlers or environments, e.g.

```json
"exports": {
  ".": {
    "import": "dist/somedom.mjs",
    "require": "dist/somedom.js",
    "browser": "dist/somedom.umd.js"
  },
  "./ssr": {
    "import": "dist/index.ssr.mjs",
    "require": "dist/index.ssr.js"
  }
}
```

This configuration yields `The package target "dist/somedom.mjs" is invalid` if consumed on other modules.

Once I prefixed the paths to `./dist` then it worked without issues.

I think this is an issue, because I ran `publint` and it said it was correct, which was not true.

So much thanks for this amazing tool, see you!

**Comment by bluwy** [comment]
Good idea! I think that's required by nodejs too so we should probably enforce this.

**Comment by bluwy** [comment]
Supported in v0.1.6. The site should be updated too.
